### PR TITLE
Fix nullable with all of

### PR DESCRIPTION
**Pull request by vsouz4**

### CODE REVIEW

**Comment by vsouz4** [comment]
```
      allOf:
        - $ref: '#/components/schemas/MemberPaymentMethod'
        - type: object
          nullable: true
```
something like this wasn't being parsed correctly (allOf was just merging properties, not the field attributes, so I added the nullable check at least to support this)

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [9.1.2] - 2022.06.06
+### Fixed
```
Comment (pelyib) [inline CHANGELOG.md:8]: It is a missing feature not a broken one, isn't it?
For me it is "added" and minor version bump

File: src/Input/Factory/FieldFactory.php [hunk 2]
```diff
@@ -202,4 +203,21 @@ private function mergeAllOfProperties(string $operationName, SpecObjectInterface
 
         return array_merge([], ...$allOfProperties);
     }
+
+    private function mergeAllOfAttributes(SpecObjectInterface $schema): SpecObjectInterface
```
Comment (pelyib) [inline src/Input/Factory/FieldFactory.php:207]: I'm not sure this function name is the best but i don't have  any idea yet.

File: CHANGELOG.md (hunk 1 above)
Comment (vsouz4) [reply CHANGELOG.md]: good point, fixed

**Pull request merged**